### PR TITLE
ES: Interfaz

### PR DESCRIPTION
**Pull request by NozomiRem**

### CODE REVIEW

File: Dialogue/Mid-Mission/co_011257.csv [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+text07#0,"""Esto explicará acerca de la caida de objetos<br><br>Los enemigos que aparecen en los Niveles, se replican de forma artificial<br><br>Debido a eso, los objetos son muy difíciles de replicar, y ellos son inestables cuando aparecen.<br><br>Ese tipo de elementos aparecen con un icono especial."""
+text08#0,"""Los objetos que caen con ese icono especial, no entran en su inventario cuando se recogen. Dado que sus datos son inestables, se almacenan como Objetos virtuales en un almacen especial.<br><br><yellow>Los objetos virtuales almacenados en este almacen especial desaparecerán cuando regrese a Campship. Solamente cuando usted termina una Mision, puede usted tomar un número fijo de ellas en la terminal extreme.<c>"""
+text09#0,"""Esto explicará sobre las órdenes de los Escenario.<br><br>Durante la Extreme Quests, le daremos órdenes experimentales durante cada Escenario, conocida como "Stage Order".<br><br>Al cumplir con estas órdenes, aumentará el número de objetos que los enemigos botan cuando los derrotas.<br><br>Además, cada vez que cumpla una cierta cantidad de ordenes, la cantidad de objetos que puede llevar aumentara cuando complete la mision."""
```
Comment (alama) [inline Dialogue/Mid-Mission/co_011257.csv]: You forget to escape quotes in `co_011257.csv'`s `text09#0`

**Comment by alama** [comment]
Learn to work in a group, make use a mergetool (like kdiff3)

I will keep fixing your mistakes as long the buildbot fails your comments

File: UI/ui_mainmode.csv [hunk 2]
```diff
@@ -134,11 +134,11 @@ BluaPivot#0,"""奥行きの調整"""
...
+Confirm_Message#0,"""Las siguientes condiciones entrarán en vigor<br>Mientras estes en un bloque compartido.<br>  · [Battle Warrior] clase Unica<br>  · PAs, tecnicas y habilidades seran olvidadas, Mesetas, CC y objetos no podran intercambiarse.<br>  · No tienes permitido depositar/sacar objetos del Almacen.<br>  · No podras recibir notificaciones de emergency quest.<br>  · Tus amigos, correo y symbol art seran cambiados a unos diferentes para usar en la nave compartida.<br><red> ※Todas estas condiciones regresaran a la normalidad despues de regresar de la nave compartida.<c><br><red> ※ Efecto Premium de NetCafe Los Puntos de Team, Adicional no se aplican. <c><br><red> ※ Su personaje puede aparecer en transmisiones en vivo.<c><br><br>¿Ir a la nave compartida?"""
```
Comment (alama) [inline UI/ui_mainmode.csv]: Are you sure you do not need help remove the 5 `NO-BREAK SPACE`s?

Comment (NozomiRem) [reply UI/ui_mainmode.csv]: give me a moment
I review it

Comment (NozomiRem) [reply UI/ui_mainmode.csv]: That for every external correction I have to come back to download everything

**Comment by NozomiRem** [comment]
I take a while to see the errors but I understand how to repair them
But for some reason, if they repair it does not update me and throw me error 
Deleting the whole folder is the only way to repair it T_T

**Pull request merged**